lib/stacker/version.rb

```diff
@@ -1,3 +1,3 @@
 module Stacker
-  VERSION = '0.2.1'
+  VERSION = '0.2.2'
```

Comment:
Do we want to bump the minor version instead, since #17 wasn't really a bug fix?